sites/cheerpx/src/content/docs/12-reference/00-CheerpX-Linux-create.md

```diff
@@ -67,6 +67,9 @@ const cx = await CheerpX.Linux.create({
 });
 ```
 
+> [!note]
```

Comment:
```suggestion
> [!note] Note
```

Reply:
There is another callout on the same page without title but cannot suggest a commit there.